lib/trento/application/read_models/host_checks_results_read_model.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trento.HostChecksExecutionsReadModel do
+  @moduledoc """
+  Host checks executions read model
+  """
+
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
+  @primary_key false
+  schema "hosts_checks_results" do
```

Comment:
hosts checks results or hosts checks execution?

Reply:
Between the two I'd prefer `HostsChecksExecution`.

---

Just as a discussion triggering point:
I cannot also think about the possibility of it being at the moment  just `ChecksExecution` :thinking: 
anyway it is bounded to the context of a host, right?

We can refine also on the language and see whether some extra differentiation is needed.